doc/develop/getting_started/index.rst

```diff
@@ -154,8 +154,8 @@ The current minimum required version for the main dependencies are:
 
          Therefore, we don't recommend using WSL when getting started.
 
-      These instructions must be run in a ``cmd.exe`` command prompt terminal window.
-      In modern version of Windows (10 and later) it is recommended to install the Windows Terminal
+      These instructions must be run in a ``cmd.exe`` command prompt windows. In modern version
+      of Windows (10 and later) it is recommended to install the Windows command prompt
```

Comment:
this might be incorrect IF we were trying to direct folks to install windows terminal. But if we are, I would suggest adding that as a step at the end.